fr/_tools.md

```diff
@@ -0,0 +1,270 @@
+
+Si vous utilisez un framework CSS pour votre projet, comme [Bootstrap](http://getbootstrap.com/) ou [Foundation](http://foundation.zurb.com/), il y a des chances qu'ils comprennent déjà un système de grilles, auquel cas je vous recommande de l'utiliser pour éviter d'avoir à gérer une dépendance supplémentaire.
+
+Dans le cas contraire, vous serez content de savoir qu'il existe deux gestionnaires de grilles de premier ordre, basé sur Sass&nbsp;: [Susy](http://susy.oddbird.net/) et [Singularity](http://singularity.gs/). Tous deux font bien plus que ce dont vous aurez jamais besoin, vous pouvez choisir celui que vous préférez et être sûrs qu'ils répondront à vos besoins même les plus complexes. À mon avis, Susy a une communauté plus importante, mais c'est juste mon opinion.
+
+Vous pouvez également choisir quelque chose de plus simple comme [csswizardry-grids](https://github.com/csswizardry/csswizardry-grids). Au bout du compte, votre choix n'aura pas beaucoup d'impact sur votre style de code, donc c'est vraiment selon vos goûts et besoins.
+
+
+
+
+### Lectures complémentaires
+
+* [Singularity](http://singularity.gs/)
+* [Singularity: Grids Without Limits](http://fourword.fourkitchens.com/article/singularity-grids-without-limits)
+* [Singularity Grid System](http://www.mediacurrent.com/blog/singularity-grid-system)
+* [Susy](http://susy.oddbird.net/)
+* [Build Web Layouts Easily with Susy](http://css-tricks.com/build-web-layouts-easily-susy/)
+* [A Complete Tutorial to Susy 2](http://www.zell-weekeat.com/susy2-tutorial/)
+* [Sass Grids: From Neat to Susy](http://www.sitepoint.com/sass-grids-neat-susy/)
+* [Bootstrap's Grid System vs Susy: a Comparison](http://www.sitepoint.com/bootstraps-grid-system-vs-susy-comparison/)
+* [How to Use Susy: Superpowered Sass Grids](http://webdesign.tutsplus.com/tutorials/how-to-use-susy-superpowered-sass-grids--cms-22744)
+* [A Creative Grid System with Sass and calc()](http://www.sitepoint.com/creative-grid-system-sass-calc/)
+
+
+
+
+
+
+## SCSS-lint
+
+Il est important de faire l'analyse statique (*lint*) de votre code. Si vous suivez les recommandations d'un guide de style, vous réduirez grandement le nombre d'erreurs mais nul n'est parfait et on peut toujours s'améliorer. On peut donc dire que faire l'analyse statique du code est aussi important que le commenter. 
+
+
+[SCSS-lint](https://github.com/causes/scss-lint) est un outil qui vous aide à conserver des fichiers SCSS propres et lisibles. Il est entièrement personnalisable et facile à intégrer à vos outils de développement.
+
+Par chance, les recommandations de SCSS-lint sont très similaires à celles de ce document. Si vous voulez configurer SCSS-lint selon les recommandations décrites ici, je vous propose la configuration suivante&nbsp;:
+
+{% highlight yaml %}
+# For SCSS-Lint v0.32.0
```

Comment:
Please, remove this line and put it at the end of the previous sentence such as:

> je vous propose la configuration suivante (SCSS-Lint v0.32.0)&nbsp;: